.cloud-v/cloud-v-pipeline

```diff
@@ -0,0 +1,55 @@
+// All CI build results of this pipeline are available at: https://dash.cloud-v.co/blue/organizations/jenkins/simde/activity
+node('J-TESTVM-1') {
+    stage('Clean Workspace') {
+        cleanWs()
+    }
+    stage('Clone') {
+        checkout scm
+    }
+    stage('Run system_info') {
+        sh '''#!/bin/bash
+            cat /proc/cpuinfo
+        '''
+    }
+    stage('Run configure') {
+        sh '''#!/bin/bash -l
+            module load riscv64-gnu-glibc/14.1.1
+            riscv64-unknown-linux-gnu-gcc --version
+            meson setup build --cross-file=./docker/cross-files/riscv64-gcc-13-unknown.cross -Db_lundef=false
+        '''
```

Comment:
Does it need to be modified here? 13->14 ?

Reply:
The file could be renamed, but its contents don't mention the GCC version

https://github.com/simd-everywhere/simde/pull/1184/files#diff-8e5bdcf98aed37c1bff927573747ea9badeb2ae4ae8f5158aeae9abffd5baff6